doc/user_guide/getting_going.rst

```diff
@@ -9,7 +9,8 @@ Download
 The following instructions are intended for a PSyclone user who wants
 to work with a released version of the code. If you are a developer or
 wish to test a specific branch of PSyclone from the GitHub repository
-please see :ref:`dev-installation` in the :ref:`developers-guide`.
+please see Developer's Set-up section in the
```

Comment:
I appreciate that these links were giving errors but I *think* that's because the stable version of the developers guide does not yet contain the dev-installation section. (This should be auto-magically picked up using the `intersphinx_mapping` entry in `conf.py`.) I've just gone back and looked at the actual error message that we get (on master) and see that it's only the second part of this reference that's the problem. I propose that we therefore keep the first one and use your replacement for the second one.  (And the same again at lines 295/296.

After some googling, I also think we should use "Developer Guide" rather than get confused over whether we should have "s" with or without an apostrophe :-)

Reply:
Done